ui/src/themes/ligera.js

```diff
@@ -206,6 +206,10 @@ export default {
         borderBottom: '1px solid #1d1d1d',
         padding: '10px !important',
         color: '#656565 !important',
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        maxWidth: '10em',
```

Comment:
I'd like to have a more "grid" styling instead of what we have now. But setting one maxWidth for all cells may not be the best approach. Or else we can end up with things like this:
<img width="1717" alt="Screen Shot 2021-04-15 at 8 25 47 PM" src="https://user-images.githubusercontent.com/331353/114954587-ee5ec200-9e28-11eb-9437-88e489687709.png">
(this is the largest Artist name in the list, so no need to truncate the title name)

Also, we would probably want to show the full name on hover. Maybe the best approach isto create our own common/Datagrid.js and override this behaviour there

Reply:
@deluan, how about having a popover on hover with Album details. Similar to Gmail, when we hover over the mail id,
![image](https://user-images.githubusercontent.com/61188295/115134989-a612f000-a032-11eb-83d6-cdf60f2146bb.png)
 
Feedbacks are most welcomed. Thank you!

Reply:
@deluan, is not it a good idea?